apps/site/components/Downloads/DownloadReleasesTable/index.tsx

```diff
@@ -6,6 +6,22 @@ import FormattedTime from '#site/components/Common/FormattedTime';
 import DetailsButton from '#site/components/Downloads/DownloadReleasesTable/DetailsButton';
 import getReleaseData from '#site/next-data/releaseData';
 
+const BADGE_KIND_MAP = {
+  'End-of-life': 'warning',
+  Maintenance: 'neutral',
+  LTS: 'info',
+  Current: 'default',
+  Pending: 'default',
+} as const;
+
+const BADGE_TEXT_MAP = {
+  'End-of-life': 'End-of-Life (EOL)',
+  Maintenance: 'Maintenance LTS',
+  LTS: 'Active LTS',
+  Current: 'Current',
+  Pending: 'Pending',
+} as const;
```

Comment:
nit: Is there a better place to store these constants?

nit 2: I'm fine with the names as is, why the changes?

Reply:
Color matching and better descriptive text, no?

Reply:
I think this file is ok to store'em as they are explicitly related to this table. If their use-case becomes wider, we can move it elsewhere

Reply:
I've opened a couple issues and PRs to that effect with the release repo to attain clarity